.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,50 +1,36 @@
-<!--- Before submitting your pull request, --->
-<!--- please complete as much as practicable of the following checklist: --->
+<!--- Type an x in the box [x] before submitting --->
 
-### Pull Request Checklist
+### Submission Checklist
```

Comment:
I think this subtitle is not relevant anymore.

Reply:
:+1: